src/scss/pages/landing/services.scss

```diff
@@ -78,9 +80,12 @@
   }
 
   @media only screen and (max-width: 768px) {
+    .section__title{
+      font-size: 3em;
+      }
     &__title {
       margin: 24px 0;
-    }
+     }
```

Comment:
this indentation looks wrong, doesn't it?

Reply:
can you type something in answers if you decide to skip my comments? I can't see any feedback